core/src/main/java/edu/wpi/grip/core/operations/opencv/NewSizeOperation.java

```diff
@@ -41,6 +41,8 @@ public NewSizeOperation(InputSocket.Factory inputSocketFactory,
     this.heightSocket = inputSocketFactory.create(heightHint);
 
     this.outputSocket = outputSocketFactory.create(outputHint);
+    
+    this.perform();//So initial value is correct.
```

Comment:
As far as I can tell SocketHint.Builder is package private scope. So I would need to add a method to SocketHints.Outputs to call SocketHint.Builder. Right?